modin/core/storage_formats/pandas/parsers.py

```diff
@@ -787,17 +787,14 @@ class PandasSQLParser(PandasParser):
     )
     def parse(sql, con, index_col, read_sql_engine, **kwargs):
 
-        enable_cx = False
         if read_sql_engine == "Connectorx":
             try:
                 import connectorx as cx
-
-                enable_cx = True
             except ImportError:
                 warnings.warn(
                     "Switch to 'pandas.read_sql' since 'connectorx' is not installed, please run 'pip install connectorx'."
                 )
-
+        enable_cx = "cx" in locals()
```

Comment:
I actually like original code more...

Reply:
Yeah, I can change this one back. I was hoping that this would appease CodeQL but that didn't end up happening.